bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/Triboson search/Triboson_M_R/Triboson_M_R_customizecards.dat

```diff
@@ -0,0 +1,7 @@
+#set param_card mass 9000024 <MASS>
```

Comment:
can you please remove the commented lines?

Reply:
OK，I removed them just now.